frontend/src/components/EasyModal.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import { RefObject } from "react";
+import RightCloseButton from "./RightCloseButton";
+import BaseModal from "./BaseModal";
+import PrimaryButton from "./PrimaryButton";
+import { Check } from "react-feather";
+
+export interface EasyModalProps {
+  children: React.ReactNode;
+  onClose?: () => void;
+  onSave?: () => void;
+  modalRef: RefObject<HTMLDialogElement>;
+  showCloseButton?: boolean;
+  title: string;
+}
+
+function EasyModal(props: EasyModalProps) {
+  const { children, onClose, onSave, modalRef, showCloseButton, title } = props;
+
+  const dialogElement = modalRef?.current;
+
+  function handleClose() {
+    onClose?.();
+    dialogElement?.close();
+  }
+
+  function handleSave() {
+    onSave?.();
+    dialogElement?.close();
+  }
+
+  return (
+    <BaseModal modalRef={modalRef}>
+      <div className="w-[299px] md:inset-0 h-[calc(100%-1rem)] max-h-full relative max-w-2xl">
+        <div className="w-full h-10 justify-between items-center inline-flex">
+          <div className="p-2 text-zinc-800 text-xl font-normal leading-normal">
+            <h2>{title}</h2>
+          </div>
+          {showCloseButton && (
+            <div className="p-2 rounded-lg justify-center items-center gap-2 flex">
+              <RightCloseButton onClick={handleClose} />
+            </div>
+          )}
+        </div>
+
+        <div className="p-2 md:p-5 space-y-4 z-10">{children}</div>
+
+        <div className="p-2 md:p-5 space-y-4 z-10">
+          <PrimaryButton onClick={handleSave} fullWidth>
+            <Check className="text-white" size={24} />
+            <span className="text-white text-sm">Lagre</span>
+          </PrimaryButton>
+        </div>
+      </div>
+    </BaseModal>
```

Comment:
Denne kan også forenkles litt. Jeg tror vi må være forsiktig med å bruke tailwind fra Figma, for den blir ofte ikke helt ritkig. Er ikke sikkert du har gjort det her, men vet du snakket om det tidligere. 😅 For eksempel bruker den her text-sm og text-xl istedenfor å bruke fontene våre som er definert i global.css. Jeg er usikker på hvilken funksjon `md:inset-0 h-[calc(100%-1rem)] max-h-full relative max-w-2xl` har her, for det skjedde ingenting da jeg fjernet disse. Her er et forslag på forenkling som er litt mer likt designet i Figma 😄:

```suggestion
    <BaseModal modalRef={modalRef}>
      <div className="w-[332px] relative flex flex-col gap-8">
        <div className="flex flex-col gap-6">
          <div className="flex flex-row justify-between items-center">
            <h2>{title}</h2>
            {showCloseButton && <RightCloseButton onClick={handleClose} />}
          </div>
          {children}
        </div>
        <PrimaryButton onClick={handleSave} fullWidth>
          <span className="text-white normal-semibold">Legg til</span>
        </PrimaryButton>
      </div>
    </BaseModal>
```